pyproject.toml

```diff
@@ -28,10 +28,10 @@ pre-commit = "^2.10.1"
 pytest = "^6.2.2"
 pytest-asyncio = "^0.15"
 black = {version = "^21.7b0", allow-prereleases = true}
+cruft = "^2.9.0"
```

Comment:
Why are we pinning versions? Is this for a specific reason? Thought we were trying to move away from this?

Reply:
This is restricting to a major version (which is what we want)